### PR TITLE
FR5610 make shortcutmapper cols resizable

### PR DESCRIPTION
**Pull request by eljefe7000**
Issue #5610

### CODE REVIEW

File: PowerEditor/src/ScitillaComponent/Buffer.cpp [hunk 1]
```diff
@@ -1381,6 +1381,7 @@ bool FileManager::loadFileData(Document doc, const TCHAR * filename, char* data,
 			{
 				if (fileFormat._encoding == SC_CP_UTF8)
 				{
+					_pscratchTilla->execute(SCI_SETCODEPAGE, SC_CP_UTF8);
```
Comment (chcg) [inline PowerEditor/src/ScitillaComponent/Buffer.cpp:1384]: @eljefe7000 Seems this is a unrelated change and shouldn't be part of this PR.

File: PowerEditor/src/WinControls/Grid/BabyGrid.cpp [hunk 2]
```diff
@@ -1725,7 +1730,7 @@ LRESULT CALLBACK GridProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
                        //if there are \n codes in the generic_string, find the longest line
                        longestline=FindLongestLine(hdc,(TCHAR*)lParam,&size);
                        //GetTextExtentPoint32(hdc,(TCHAR*)lParam,lstrlen((TCHAR*)lParam),&size);
-                       required_width = longestline+15;
+					   required_width = longestline+15;
```
Comment (chcg) [inline PowerEditor/src/WinControls/Grid/BabyGrid.cpp:1733]: @eljefe7000 Whitespace changes here and below shouldn't be part of a commit. Was this intended?

Comment (eljefe7000) [reply PowerEditor/src/WinControls/Grid/BabyGrid.cpp:1733]: @chcg my apologies.  It was not intended.  Not sure how that got in there.  My code shows it aligned correctly with the previous line.

File: PowerEditor/src/ScitillaComponent/Buffer.cpp (hunk 1 above)
Comment (donho) [reply PowerEditor/src/ScitillaComponent/Buffer.cpp:1384]: @eljefe7000  yes, @chcg  is right. Why this line?

File: PowerEditor/src/ScitillaComponent/Buffer.cpp [hunk 3]
```diff
@@ -1398,6 +1399,7 @@ bool FileManager::loadFileData(Document doc, const TCHAR * filename, char* data,
 			else
 			{
 				lenConvert = unicodeConvertor->convert(data, lenFile);
+				_pscratchTilla->execute(SCI_SETCODEPAGE, CP_ACP);
```
Comment (donho) [inline PowerEditor/src/ScitillaComponent/Buffer.cpp:1402]: This line has nothing to do with this PR either.

**Comment by donho** [comment]
This PR contains the modification for PR #8077
Please verify your commit before doing the PR.

**Comment by eljefe7000** [comment]
I'm sorry for getting the two requests combined.  After I made a fix for
the encoding issue, I created a branch and made a fix for the column resize
issue.  The branch must've included the encoding fix.  I will try to get
these two separated and resubmit pull requests.
